src/main/openapi/ckan.yaml

```diff
@@ -164,6 +164,10 @@ components:
           items:
             $ref: "#/components/schemas/CkanContactPoint"
           title: Contacts
+        creators:
+          type: array
+          items:
+            $ref: "#/components/schemas/Creator"
```

Comment:
important: please add `Ckan` to class name to avoid name clashing, if we decide to support other metadata sources.

Reply:
Great catch, I added the prefix in the latest revision.